components/workflow-mgt/org.wso2.carbon.identity.workflow.mgt/src/main/java/org/wso2/carbon/identity/workflow/mgt/dao/WorkflowRequestDAO.java

```diff
@@ -126,6 +126,54 @@ public WorkflowRequest retrieveWorkflow(String uuid) throws InternalWorkflowExce
         return null;
     }
 
+    public static String retrieveExternalWorkflowId(String workflowId) throws InternalWorkflowException {
+
+        Connection connection = IdentityDatabaseUtil.getDBConnection(false);
+        PreparedStatement prepStmt = null;
+        ResultSet rs = null;
+
+        String query = SQLConstants.GET_WORKFLOW_ID_QUERY;
+        try {
+            prepStmt = connection.prepareStatement(query);
+            prepStmt.setString(1, workflowId);
+            rs = prepStmt.executeQuery();
+            if (rs.next()) {
+                return rs.getString(SQLConstants.WORKFLOWID_COLUMN);
+            }
+        } catch (SQLException e) {
+
```

Comment:
Remove unnecessary new lines before and after line 145.

Reply:
addressed the comment ref:https://github.com/wso2/carbon-identity-framework/pull/4519/commits/ce9bfa08202eb691d2a4f87aebd30fe3445f0fd5